php-8.3.yaml

```diff
@@ -315,7 +306,10 @@ subpackages:
             echo 'decorate_workers_output = no'; \
           } | tee ${{targets.subpkgdir}}/etc/php/php-fpm.d/zz-apko.conf
 
-# The release candidates are in a different location, so we can't really
-# monitor the github tags for these since they differ.
 update:
-  enabled: false
+  enabled: true
+  github:
```

Comment:
Do we not need release monitor here? as we're not pulling from github?

Reply:
Good question, we had set up the other php this way, and I also checked with the wolfictl update. Maybe we should instead move them all to use github to checkout instead of the distributions? 🤔 
I think we want to use github for update over release monitoring, so I think the right thing is to see about pulling the source from github, yeah. Let me see...

Reply:
Interesting, looks like it now wants re2c which we do not currently seem to have? 🤔 
```
⚠️  aarch64   | configure: error: re2c 1.0.3 is required to generate PHP lexers.
```

Reply:
I guess we meant to switch over to git checkout in this previous commit? https://github.com/wolfi-dev/os/pull/5191/files when we setup git polling?